README.md

```diff
@@ -66,13 +63,13 @@ This module performs assignment of new samples into an existing clustering which
 
 #Generate pairwise distance file
 ```
-  profile_dists -q ./example/call/query.profile.txt -r ./example/call/ref.profile.txt --outfmt pairwise --distm hamming -o ./example/call/hamming
+  profile_dists -q ./example/call/query.profile.txt -r ./example/call/ref.profile.txt --distm hamming -o ./example/call/hamming
```

Comment:
I like agree that removal of redundant `--outfmt pairwise` is justified as anyways the output is a matrix which is displays pairwise distances anyways.

Reply:
Wait.. maybe I shouldn't have changed this since  it's a `profile_dists` command and not a GAS command... Haha.

Reply:
Reverted here (as I did not change `profile_dists`): 2a204c2